main.go

```diff
@@ -240,6 +251,13 @@ func initFlags(fs *pflag.FlagSet) {
 		string(options.PowerVSProviderIDFormatV1),
 		"ProviderID format is used set the Provider ID format for Machine",
 	)
+
+	fs.StringVar(
+		&endpoints.PowerVSServiceEndpoints,
+		"powervs-service-endpoints",
```

Comment:
why not use the `service-endpoints` because we are not going to have just powervs, lets make all the necessary changes like aws implementation

Reply:
I was thinking of keeping both independent from one another. We can go with single service-endpoints approach something like `service-endpoints=https://us-south-stage01.iaasdev.cloud.ibm.com/v1;https://dal.power-iaas.test.cloud.ibm.com,https://resource-controller.test.cloud.ibm.com` with vpc and pas endpoints separated by `;`


sounds okay?

Reply:
how about this:
```
vpc=https://us-south-stage01.iaasdev.cloud.ibm.com/v1;powervs=https://dal.power-iaas.test.cloud.ibm.com;rc=https://resource-controller.test.cloud.ibm.com
```

so that it becomes very easy for user to decide what they want to pass